### PR TITLE
docs: make docs about sort and transforms clearer

### PR DESCRIPTION
**Pull request by Crayon-ShinChan**
[[中文版模板 / Chinese template](https://github.com/VisActor/VChart/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Release
- [x] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

File: docs/assets/option/en/common/data.md [hunk 1]
```diff
@@ -138,12 +138,14 @@ const spec = {
   ]
 }
 // 最终数据
+// 数据会按照 date 在原数据中出现的顺序进行排序
```
Comment (xile611) [inline docs/assets/option/en/common/data.md:141]: 需要改成英语的注释

File: docs/assets/option/en/common/data.md [hunk 2]
```diff
@@ -267,11 +269,12 @@ const spec = {
 // 最终数据
```
Comment (xile611) [inline docs/assets/option/en/common/data.md:269]: 需要改成英语的注释

File: docs/assets/option/en/common/data.md (hunk 1 above)
Comment (Crayon-ShinChan) [reply docs/assets/option/en/common/data.md]: done

File: docs/assets/option/en/common/data.md (hunk 2 above)
Comment (Crayon-ShinChan) [reply docs/assets/option/en/common/data.md]: done

**Review by xile611 — APPROVED** [review]

**Pull request merged**